dockers/docker-lldp-sv2/lldpmgrd

```diff
@@ -208,32 +210,43 @@ class LldpManager(object):
         sel.addSelectable(sst_appdb)
 
         # Listen for changes to the PORT table in the CONFIG_DB and APP_DB
+        # Only when port is present in both DBs and the operation status is UP
+        # we are going to configure it in lldp
         while True:
             (state, c) = sel.select(SELECT_TIMEOUT_MS)
 
             if state == swsscommon.Select.OBJECT:
+
+                # handle alias or description changes, by listening to
+                # changes in config DB
                 (key, op, fvp) = sst_confdb.pop()
-                if fvp:
-                    fvp_dict = dict(fvp)
+                if op == "DEL":
+                    self.config_port_names.discard(key)
```

Comment:
Do we need do "lldpcli unconfigure" for the ports that are removed?

Reply:
I don't see any "lldpcli unconfigure" command for the port. Could you please point me to it? Also, what would be the result of that command?

Reply:
The question was whether we should do lldp unconfigure for the properties on the ports we had configured,  if all the configurations on ports are set only and could not be unconfigured, then we can leave it as is in the PR. 
looks like "lldp portidsubtype" and "description" etc can only be set,  so we can leave it as is.